test/uk/gov/hmrc/cipphonenumberverification/connectors/GovUkConnectorSpec.scala

```diff
@@ -75,7 +89,7 @@ class GovUkConnectorSpec extends AnyWordSpec
               "template_id": "template-id-fake",
               "personalisation": {
                 "clientServiceName": "cip-phone-service",
-                "passcode": "test",
+                "passCode": "test",
```

Comment:
passcode is one word, template has been updated

Reply:
ok, quite a small change 😒

Reply:
yeah, just want to make it consistent with email service, plus I have already updated the templates in govuk notify

Reply:
done